src/framework/operations.hpp

```diff
@@ -287,7 +610,8 @@ struct Op {
   bool conditional = false; // is gate conditional gate
   uint_t conditional_reg; // (opt) the (single) register location to look up for
                           // conditional
-  RegComparison bfunc;    // (opt) boolean function relation
+  BinaryOp binary_op;     // (opt) boolean function relation
```

Comment:
`Op.bfunc` is replaced with `Op.binary_op` but `OpTyp::bfunc` is still remaining as is. Is it OK?

Reply:
Thanks. It should be `binary_op`.

Reply:
Hmmm, we still use `Operations::OpType::bfunc` and keeping `apply_bfunc` is reasonable.